cranelift/codegen/src/isa/riscv64/inst/mod.rs

```diff
@@ -1637,23 +1696,29 @@ impl Inst {
                 eew,
                 to,
                 from,
+                ref mask,
                 ref vstate,
                 ..
             } => {
                 let base = format_vec_amode(from, allocs);
                 let vd = format_reg(to.to_reg(), allocs);
-                format!("vl{}.v {},{} {}", eew, vd, base, vstate)
+                let mask = format_mask(mask, allocs);
+
+                format!("vl{}.v {},{}{} {}", eew, vd, base, mask, vstate)
             }
             Inst::VecStore {
                 eew,
                 to,
                 from,
+                ref mask,
                 ref vstate,
                 ..
             } => {
                 let dst = format_vec_amode(to, allocs);
                 let vs3 = format_reg(*from, allocs);
-                format!("vs{}.v {},{} {}", eew, vs3, dst, vstate)
+                let mask = format_mask(mask, allocs);
+
+                format!("vs{}.v {},{}{} {}", eew, vs3, dst, mask, vstate)
```

Comment:
Drive-by comment on this, not something specific to this PR, but nowadays `format!("{foo}")` support in Rust can probably help simplify a lot of these format strings. This affects all backends, not just risc-v, but may be worth incrementally updating from now on if you're interested.

Reply:
Yeah, sounds good, I'll start using that.